pandas/tests/window/test_groupby.py

```diff
@@ -275,8 +273,8 @@ def test_groupby_rolling_center_on(self):
         )
         result = (
             df.groupby("gb")
-            .rolling(6, on="Date", center=True, min_periods=1)
-            .value.mean()
+                .rolling(6, on="Date", center=True, min_periods=1)
```

Comment:
can you not change unrelated things

Reply:
pls don't apply your settings otherwise this will not pass pre-commit.

Reply:
Sorry. I removed the autoformatted changes..